daphne/src/roles/aggregator.rs

```diff
@@ -26,12 +24,10 @@ use crate::{
 pub trait DapReportInitializer {
     /// Initialize a sequence of reports that are in the "consumed" state by initializing VDAF
     /// preparation.
-    async fn initialize_reports<'req>(
+    async fn initialize_reports(
         &self,
         is_leader: bool,
```

Comment:
I want to eventually get rid of these `is_leader: bool` parameters. I know they closely model the draft but I think we should use `DapRole` enum instead

Reply:
SGTM, but let's save for a future PR.